libnavui-androidauto/unreleased/bugfixes/changes.md

```diff
@@ -0,0 +1 @@
+- Fixed PlacesListOnMapScreen issues when screen changed cause places to disappear
```

Comment:
The directory should be `libnavui-androidauto/changelog/unreleased/bugfixes/` instead of `libnavui-androidauto/unreleased/bugfixes/`. You can use the script to generate the changelog:
```bash
cd libnavui-androidauto
python3 ../scripts/changelog/add_changelog.py -b "Fixed PlacesListOnMapScreen issues when screen changed cause places to disappear."
```